crates/notion-core/src/version/mod.rs

```diff
@@ -10,7 +10,7 @@ use notion_fail::{Fallible, ResultExt};
 
 use self::serial::parse_requirements;
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
Is there a need for the `PartialEq, Eq, PartialOrd, Ord` derivations on `VersionSpec`? I don't see any other changes that would require comparing / sorting them. Not that it's a big issue either way, just curious

Reply:
Yes: because `VersionSpec`s are used in `ToolSpec`s. For a type to implement these traits, the types it depends on must *also* implement them.

Reply:
Ah, makes perfect sense, thanks!